views/sapiV1CapiV2Experiments/basic1.hbs

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<style>
+			.grouping {
+				font-weight:bold;
+				background-color: coral;
+			}
+			.blank_row {
+			    height: 30px !important; /* overwrites any other rules */
+			    background-color: #FFFFFF;
+			}
+			.name {
+				font-size: 150%;
+				/* white-space: nowrap; */
+			}
+			.articles {
+				width: 60%;
+				padding: 0;
+				margin: 0;
+				list-style: none;
+			}
+			.articles65 {
+				width: 65%;
+			}
+			.thumbnail {
+				width: 30%;
+			}
+			.thumbnails {
+				width: 25%;
+			}
+			.pubdate {
+				font-size: 75%;
+				font-style: italic;
+			}
+
+		</style>
+
+	</head>
+	<body>
+		<h1>SapiV1CapiV2 experiments: correlations in article annotations</h1>
+		<h3>based on {{context.numArticles}} articles, of which {{context.numArticlesInGenres}} articles in genres "{{context.genresString}}"</h3>
```

Comment:
Please use semantic heading numbering. `<h1>` should be followed by `<h2>`

Reply:
done